kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerContainerManager.java

```diff
@@ -264,6 +264,16 @@ private ContainerConfiguration createContainerConfiguration() {
         if (!this.options.getBackendsConfigs().isEmpty()) {
             this.options.getBackendsConfigs().forEach(config -> entrypointOverride.add("--backend-config=" + config));
         }
+        entrypointOverride.add("--allow-metrics=" + this.options.areMetricsEnabled());
+        entrypointOverride.add("--allow-gpu-metrics=" + this.options.areGpuMetricsEnabled());
+        entrypointOverride.add("--allow-cpu-metrics=" + this.options.areCpuMetricsEnabled());
```

Comment:
As I said above I would try to reduce the number of user configurable parameters:

```suggestion
        entrypointOverride.add("--allow-metrics=" + this.options.areMetricsEnabled());
        entrypointOverride.add("--allow-gpu-metrics=" + this.options.areMetricsEnabled());
        entrypointOverride.add("--allow-cpu-metrics=" + this.options.areMetricsEnabled());
```